DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DSACampaignsHangoutsServlet.java

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 Google LLC
```

Comment:
Since all the code in the DSACampaignsHangoutsServlet.java file is copied from the other DSA campaigns servlet, can you remove the repeated code and make functions with the shared code that both servlets can call?

Reply:
This is also important because in the DSA campaigns servlet, I added the following line: PendingCampaignsExistServlet.changePendingCampaignsExistStatus(1);
which is required now when posting DSA campaigns for the estimations to run.

Reply:
Resolved